### DISCUSSION TITLE
Old version?

### DISCUSSION THREAD

**Discussion by bigdaddy2142** [pierrelasse/YimStuff#14] [Q&A]
<img width="960" alt="v" src="https://github.com/pierrelasse/YimStuff/assets/153862933/c9767b08-0951-4259-a728-02b5aee87166">

**Comment by pierrelasse** [comment]
14 hours? Might wanna fix this and put to a week or something.

**Reply by bigdaddy2142** [reply]
So its not actually old?

**Reply by pierrelasse** [reply]
Define old. Your version is technically older than "now". 😃😃

I added this to let the user know that your version is old and might be outdated since i cant access the internet to check if there is a new version.

I need to correct the time calculation to resolve this issue.
Edit: Maybe there could also be a button to hide the notification.
